src/main/webapp/app/service/firebase/firebase-gene-service.ts

```diff
@@ -291,7 +292,7 @@ export class FirebaseGeneService {
         hugoSymbol,
         tumor.excludedCancerTypes_uuid,
         !excludedCancerTypesReview.isChangeReverted,
-        false
+        false,
```

Comment:
Did prettier add this? If so, why wasn't it do that before?

Reply:
Yeah, prettier added that comma after I removed it and git commited. Not sure why it wasn't added before.